catalog/transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -97,6 +99,17 @@
 
     private MetacardType metacardType = null;
 
+    private static final Map<String, String> MIME_TYPE_DATA_TYPE_MAP;
+
+    static {
+        MIME_TYPE_DATA_TYPE_MAP = new HashMap<>();
+        MIME_TYPE_DATA_TYPE_MAP.put("image", "Image");
```

Comment:
may be able to use this guava class for these constants http://google.github.io/guava/releases/19.0/api/docs/com/google/common/net/MediaType.html